packages/api2/src/praise/services/praise.service.ts

```diff
@@ -434,4 +437,170 @@ export class PraiseService {
 
     return assignedPraiseCount;
   };
+
+  /**
+   * Creates praises with a given receiver and reason
+   *  and returns the created praises
+   *
+   * @param {PraiseCreateInputDto} data
+   * @returns {Promise<PraiseCreateResponseDto>}
+   * @throws {ServiceException}
+   */
+  createPraiseItem = async (
```

Comment:
Handling of forwarder info needs to be here as well so we can use same endpoint for `/forward`.

Reply:
Ok, i think I fixed all other comments except this one.. it should be easy but I want to check with you first to be sure what needs to be done.